store/storetest/channel_store.go

```diff
@@ -5173,8 +5173,8 @@ func testChannelStoreSearchInTeam(t *testing.T, ss store.Store) {
 	}
 
 	for name, search := range map[string]func(teamId string, term string, includeDeleted bool) (*model.ChannelList, error){
-		"AutocompleteInTeam": ss.Channel().AutocompleteInTeam,
-		"SearchInTeam":       ss.Channel().SearchInTeam,
+		// "AutocompleteInTeam": ss.Channel().AutocompleteInTeam,
+		"SearchInTeam": ss.Channel().SearchInTeam,
```

Comment:
Hi, for SearchInTeam, did we need to add userId too ?

Reply:
cc: @agnivade

Reply:
I don't believe you need to add in `SearchInTeam`. You can use the same function for `AutocompleteInTeam`, just modify the test case to include private channels too.